libraries/engage/product/components/RelationsSlider/style.js

```diff
@@ -21,3 +25,41 @@ export const card = css({
   height: '100%',
   margin: `0 ${variables.gap.small}px`,
 }).toString();
+
+export const gridCard = css({
+  height: '100%',
+  margin: 0,
+}).toString();
+
+export const showMore = css({
+  position: 'absolute',
+  display: 'block',
+  padding: `${variables.gap.xsmall}px ${variables.gap.small * 1.5}px`,
+  top: 0,
+  right: variables.gap.big,
+  fontSize: '0.875rem',
+  lineHeight: 1.1,
+  borderRadius: 4,
+  border: `1px solid ${colors.shade5}`,
+});
+
+export const sheet = css({
+  maxHeight: `calc(100vh - ${variables.navigator.height}px)`,
```

Comment:
The z-index of the ToastDrawer is lower, that the one of the sheet. That causes, that when one removes a related product from the list via the sheet, the message to revert the decision isn't visible to the user.

Reply:
Well, that seems to be a bigger issue. The Toast drawer is part of the AppFooter component, which also holds the payment bar in iOS. This would need a bigger refactoring of how the SnackBar / Toaster is implemented.

Reply:
I saw that you created a ticket for that.